src/extensions/default/HealthData/main.js

```diff
@@ -58,10 +60,15 @@ define(function (require, exports, module) {
         brackets.test.HealthDataManager      = HealthDataManager;
         brackets.test.HealthDataNotification = HealthDataNotification;
         brackets.test.HealthDataPopup        = HealthDataPopup;
+
+        var prefs = PreferencesManager.getExtensionPrefs("healthData"),
+            hdPref = prefs.get("healthDataTracking");
```

Comment:
`hdPref` should have been `isHDTracking` like it has been used in `HealthDataManager.js`